hyperspy/signal_tools.py

```diff
@@ -1005,7 +1005,8 @@ def create_interpolation_line(self):
         self.interpolated_line.set_line_properties(
             color='blue',
             type='line')
-        self.signal._plot.signal_plot.add_line(self.interpolated_line)
+        self.signal._plot.signal_plot.add_line(self.interpolated_line,
+                                               update_on_navigation=False)
```

Comment:
I think that ``update_on_navigation`` may not be needed as setting its ``auto_update`` attribute to ``False`` achieves the same effect and, ideally, there should only be one way of achieving the effect.

Reply:
Yes, this is better, with a caveat that we need to ignore the `auto_update=False` on the `span_selector_changed` callback.